clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -40,6 +40,15 @@
  * This client was introduced in 0.11.0.0 and the API is still evolving. We will try to evolve the API in a compatible
  * manner, but we reserve the right to make breaking changes in minor releases, if necessary. We will update the
  * {@code InterfaceStability} annotation and this notice once the API is considered stable.
+ *
+ * <p>The following exceptions can be thrown when calling {@code get()} on the futures obtained from AdminClient methods</p>
+ * <ul>
+ *     <li>{@link org.apache.kafka.common.errors.TimeoutException}
+ *     if the request was not completed in within the given {@link CreateTopicsOptions#timeoutMs()}</li>
+ *     <li>{@link org.apache.kafka.common.errors.TopicAuthorizationException}
+ *     if the authenticated user is not authorized to alter the topic</li>
+ * </ul>
```

Comment:
You should also include UnknownServerException here, which indicates an internal problem on the server side.